src/dmd/declaration.d

```diff
@@ -1088,9 +1088,6 @@ extern (C++) class VarDeclaration : Declaration
     VarDeclaration lastVar;         // Linked list of variables for goto-skips-init detection
     uint endlinnum;                 // line number of end of scope that this var lives in
 
-    /// it was declared in an anon union
-    bool isAnonUnionMember;
```

Comment:
C++ header needs to be reverted too.

Reply:
Yes i'm about to force push but got a little issue here...